index.html

```diff
@@ -5,13 +5,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Place favicon below when found -->
-    <link rel="favicon" href="#" type="image/x-icon">
+    <link rel="icon" href="assets/sunfavicon.png"/>
     <link rel="stylesheet" href="stylesheets/styles.css"/>
     <title>Catch the Sun</title>
 </head>
 <body>
     <div class="header">
-        <p class="title">Catch the Sun</p>
+        <div class ="CatchTheSunLogo">
+        <!-- Placeholder Logo, needs styling -->
```

Comment:
A thought: 

- Could we possibly change the class name to "logo" instead of "CatchTheSunLogo" to make it short and simple?